install/homebox-install.sh

```diff
@@ -26,6 +26,52 @@ msg_info "Installing Homebox"
 RELEASE=$(curl -s https://api.github.com/repos/sysadminsmedia/homebox/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
 wget -qO- https://github.com/sysadminsmedia/homebox/releases/download/${RELEASE}/homebox_Linux_x86_64.tar.gz | tar -xzf - -C /opt
 chmod +x /opt/homebox
+cat <<EOF >/opt/.env
```

Comment:
```suggestion
cat <<EOF >/opt/.env
HBOX_MODE=production
HBOX_WEB_PORT=7745
HBOX_WEB_HOST=0.0.0.0
EOF
```
Can we just add minimal config and people that want to configure something extra can add that themselves?

Reply:
These are just default settings, but fine to be explicit on a few to make use of the `.env` file